barretenberg/sol/bootstrap.sh

```diff
@@ -34,7 +34,7 @@ function build_sol {
         # Ensure libraries are at the correct version
         git submodule update --init --recursive ./lib
```

Comment:
was this temporary?

Reply:
it was added when this pr was first opened as the formatter could not handle the asm blocks. I think it may be fixed with newer foundries